src/lib/utils/tools/changeId.ts

```diff
@@ -1,9 +0,0 @@
-export function changeId(originalId: string): string {
```

Comment:
This is now `function getKeyRingIdFromIdToken(idToken: IdTokenWithData<any>): string`. Passing the ID token directly also helps to prevent mistakes, as you could call `cangeId` and pass it `sub` or any other `string` by mistake.